### PR TITLE
added openunison login plugin

### PR DESCRIPTION
**Pull request by mlbiam**
verified on linux, macos, and windows

### CODE REVIEW

File: plugins/oulogin.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: oulogin
+spec:
+  version: "v0.0.3"
+  homepage: https://github.com/TremoloSecurity/kubectl-login
+  shortDescription: "login to a cluster via OpenUnison from the command line"
+  description: |
+    Launches a browser to login to the cluster via OpenUnison.  OpenUnison's certificate MUST be trusted by the local system.
```
Comment (ahmetb) [inline plugins/oulogin.yaml:10]: Can you please line wrap at 80 chars

File: plugins/oulogin.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: oulogin
+spec:
+  version: "v0.0.3"
+  homepage: https://github.com/TremoloSecurity/kubectl-login
+  shortDescription: "login to a cluster via OpenUnison from the command line"
```
Comment (ahmetb) [inline plugins/oulogin.yaml:8]: please capitalize.
no need to say "from command line" as that's what plugins are.

**Comment by mlbiam** [comment]
Thanks @ahmetb updates made.  will work on automating the updates

**Comment by corneliusweig** [comment]
Looks great! Thanks for submitting to `krew-index`!
/hold cancel

**Pull request merged**